hyrax/app/indexers/publication_indexer.rb

```diff
@@ -47,6 +48,7 @@ def self.show_fields
       fields << Solrizer.solr_name('place', :stored_searchable)
       fields << Solrizer.solr_name('table_of_contents', :stored_searchable)
       fields.concat ComplexField::DateIndexer.date_show_fields
+      fields.concat ComplexField::CustomPropertyIndexer.custom_property_show_fields
```

Comment:
The corresponding file for datasets, dataset_indexer.rb, has this line for `self.search_fields` as well. I believe we should also include it for publication search as well, for consistency.